Product/SoapUI_Test/RegressionSuite/Standard/AuditLogging-Standard/AuditLogging-Standard-soapui-project.xml

```diff
@@ -4583,7 +4589,7 @@ if (propertiesFile.exists()) {
         </con:property>
         <con:property>
           <con:name>ParticipantObjectIdentification_Responder.@ParticipantObjectID</con:name>
-          <con:value>D123401^^^&amp;1.1&amp;ISO</con:value>
+          <con:value>D123402^^^&amp;2.2&amp;ISO</con:value>
```

Comment:
Will be creating a ticket for this issue per offline discussion with @alameluchidambaram and @sailajaa